src/plugins/console/public/application/models/legacy_core_editor/mode/worker/worker.js

```diff
@@ -1,5 +1,57 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/* @notice
+ *
+ * ***** BEGIN LICENSE BLOCK *****
```

Comment:
I haven't seen us use the `***** BEGIN LICENSE BLOCK *****` and `***** END LICENSE BLOCK *****` before. Does this enable something that I'm not aware of?

However, elsewhere, I've seen a pre-amble prior to the license block with text similar to the following, perhaps we should add this?:
```
This product includes code that is based on flot-charts, which was available
under a "MIT" license.
```

Reply:
Yeah I was not sure how verbatim the copy of the header in Ace's files should be. It does make sense to me that these are not required though, will remove them.

Reply:
> However, elsewhere, I've seen a pre-amble prior to the license block with text similar to the following, perhaps we should add this for consistency sake?

I removed the BEGIN and END markers and will add the pre-amble specific to the Ace editor license. Would you mind taking another look?